Apps/Encounter/src/appsettings.Development.json

```diff
@@ -1,7 +1,13 @@
 {
-    "Logging": {
-        "LogLevel": {
-            "HealthGateway": "Trace"
+    "Serilog": {
+        "MinimumLevel": {
+            "Default": "Debug",
+            "Override": {
+                "Microsoft ": "Information",
```

Comment:
There's a lot "Microsoft " references in the appsettings with an extra space in there.